al_aws.js

```diff
@@ -0,0 +1,106 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2017, Alert Logic, Inc
+ * @doc
+ *
+ * Helper class for lambda function utility and helper methods.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+
+const AWS = require('aws-sdk');
```

Comment:
@kkuzmin I would not like to split the library into two different pieces now, that is what we are trying to avoid by moving it here. Otherwise we would have to copy and paste the module in a new lambda collector. I think we can deal with it once we are sure we would like to use this piece in azure. I think it is not possible now because the way we refactored it thus we will be using it just in aws for now. If there is time to refactor it and use it in azure it will be fairly easy to move the module into something like `al-aws-collector`...

Reply:
@tomdos this is a good point. I suggest to have`al-collector-js` as a common lib to be shared between all types of collectors. And I'm all hands up to having `al-aws-collector-js` which uses this lib.

Reply:
@kkuzmin yep and let's do it once the time comes